tools/editor/plugins/script_editor_plugin.cpp

```diff
@@ -288,6 +288,7 @@ void ScriptTextEditor::_load_theme_settings() {
 
 	get_text_edit()->set_custom_bg_color(EDITOR_DEF("text_editor/background_color",Color(0,0,0,0)));
 	get_text_edit()->add_color_override("font_color",EDITOR_DEF("text_editor/text_color",Color(0,0,0)));
+	get_text_edit()->add_color_override("line_number_color",EDITOR_DEF("text_editor/line_number_color",Color(0,0,0)));
```

Comment:
The second parameter is the default value to be used if `text_editor/line_number_color` is not defined in the editor settings: [/tools/editor/editor_settings.cpp#L151-L158](https://github.com/godotengine/godot/blob/57cddf3811b3abfe557aac4ebcf55394b51ee596/tools/editor/editor_settings.cpp#L151-L158)